.devcontainer/vmr-source-build/init.sh

```diff
@@ -23,16 +25,8 @@ vmr_branch=$(git -C "$installer_dir" log --pretty=format:'%D' HEAD^ \
   | sed 's@origin/@@' \
   | sed 's@,.*@@')
 
-pushd "$installer_dir"
-  "./eng/vmr-sync.sh"      \
-    --vmr "$vmr_dir"       \
-    --tmp "$tmp_dir"       \
-    --branch "$vmr_branch" \
-    --debug
-popd
+"$workspace_dir/synchronize-vmr.sh" --branch "$vmr_branch" --debug
 
 # Run prep.sh
 unset RepositoryName
-pushd "$vmr_dir"
-./prep.sh
-popd
+sh -c "cd \"$vmr_dir\" && ./prep.sh"
```

Comment:
One shell trick you could also do when calling scripts in another directory is to call the cd and the commands in a subshell, using parentheses. In this case it'd be `(cd "$vmr_dir" && ./prep.sh)`. The CWD is affected only in the subshell (a new shell process spawned). Essentially, it's exactly the same as this explicit call to the shell (with a subtle difference of how the sh binary is found).

Reply:
I like that better, will change